content/yaml/yaml-getting-started.md

```diff
@@ -437,6 +440,19 @@ publishing:
         fi
 ```
 
+### Labels
+
+You may use `codemagic.yaml` to define several list of labels for your apps. This will allow you to white label your apps and is visible in `/builds` dashboard page and `/app/<app-id>/build/<build-id>` pages. As shown in the snippet below, labels also support environment variables as value and goes under a workflow.
```

Comment:
Nice, but I think it can be improved a little. We will first support displaying the labels and then soon after also filtering builds based on them. We'll update this part in docs when needed.
```suggestion
You may use `codemagic.yaml` to define labels for your apps. Labels serve as additional information about the workflow you are building and are helpful when you have multiple versions of a workflow, e.g. when you build white label apps. The labels are visible on the `/builds` and `/app/<app-id>/build/<build-id>` pages. As shown in the snippet below, labels also support environment variables.
```